Tiltfile

```diff
@@ -1,4 +1,4 @@
-BAZEL_RUN_CMD = "bazel run --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 %s"
```

Comment:
Why did you remove this platforms flag?

Reply:
I simply don't think it is needed. Perhaps I'm wrong. Is this plateform the only one supported?

Reply:
It's been a while since I looked but my memory is that without specifying the platform I would end up with say macOS binaries in a Linux container if I was running Tilt on macOS. Then the Linux container wouldn't be able to start them.

Of course now I don't have a macOS machine to test with so I can't be sure. @nicks can you confirm?

Reply:
@jazzdan That makes total sense. Maybe it should be mentionned in the [bazel documentation page](https://docs.tilt.dev/integrating_bazel_with_tilt.html)

Reply:
Good idea! I created an issue for that [here](https://github.com/tilt-dev/tilt.build/issues/626).

Reply:
@jazzdan ya, gmas and I'll test on my windows box

Reply:
yep, confirmed! the `--platforms` line is necessary. if you remove it, you get exec format errors because it tries to copy a macos binary to the linux container